composer.json

```diff
@@ -9,7 +9,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3",
+        "php": "^5.3.9|^7.0",
```

Comment:
Hi @webmozart , 

In this I have mentioned 5.3.9 . But do you think 5.3.3 is the right one ?